include/bluetooth/mesh/light_xyl_cli.h

```diff
@@ -0,0 +1,348 @@
+/*
+ * Copyright (c) 2020 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+ */
+
+/**
+ * @file
+ * @defgroup bt_mesh_light_xyl_cli Light xyL Client model
+ * @{
+ * @brief API for the Light xyL Client model.
+ */
+
+#ifndef BT_MESH_LIGHT_XYL_CLI_H__
+#define BT_MESH_LIGHT_XYL_CLI_H__
+
+#include <bluetooth/mesh/light_xyl.h>
+#include <bluetooth/mesh/model_types.h>
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+struct bt_mesh_light_xyl_cli;
+
+/** @def BT_MESH_LIGHT_XYL_CLI_INIT
+ *
+ * @brief Initialization parameters for a @ref bt_mesh_light_xyl_cli instance.
+ */
+#define BT_MESH_LIGHT_XYL_CLI_INIT(_handlers)                                  \
```

Comment:
```suggestion
/** @def BT_MESH_LIGHT_XYL_CLI_INIT
 *
 * @brief Initialization parameters for a @ref bt_mesh_light_xyl_cli instance.
 *
 * @sa bt_mesh_light_xyl_cli_handlers
 *
 * @param[in] _handlers Optional message handler structure.
 */
#define BT_MESH_LIGHT_XYL_CLI_INIT(_handlers)                                  \
```
I am sure that you have to document macro parameter _handlers also (https://github.com/nrfconnect/sdk-nrf/pull/3273#discussion_r515974973)